examples/backends/vllm/launch/xpu/loop_device_aware_router_vllm_3B.sh

```
@@ -0,0 +1,135 @@
+    vllm bench serve \
+        --model "$MODEL" \
+        --backend openai-chat \
+        --endpoint /v1/chat/completions \
+        --dataset-name random-mm \
+        --input-len $INPUT_LEN \
+        --output-len $OUTPUT_LEN \
+        --random-mm-base-items-per-request $NUM_IMAGES \
+        --random-mm-bucket-config "{$IMAGE_RESOLUTION: 1.0}" \
+        --seed $SEED \
+        --num-prompts $NUM_PROMPTS \
+        --request-rate $rate \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)